.pre-commit-config.yaml

```diff
@@ -1,33 +1,77 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
```

Comment:
I retained this it is helpful to explicitly link to the documentation of the syntax (first line) and the hooks used (second line), so that those coming across the file can quickly navigate to the canonical source explaining these two aspects.